test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -51,6 +56,20 @@
   @Rule
   public ExpectedException thrown = ExpectedException.none();
 
+  @Before
```

Comment:
That isn't entirely true, I'm actually moving some test data out of the proxies and into the tests. The plan is to have the test data local to each test. I'm experimenting a little with the tables. If they are globally obvious (like ACLs) I put them in `@Before`. If they are intrinsically tied to the test data, as with most of the tests, I'm keeping the tables local as well.